ccx_messaging/consumers/consumer.py

```diff
@@ -62,13 +63,7 @@ def __init__(
         publisher,
         downloader,
         engine,
-        group_id=None,
-        incoming_topic=None,
-        bootstrap_servers=None,
-        max_poll_records=None,
-        max_poll_interval_ms=None,
-        heartbeat_interval_ms=None,
-        session_timeout_ms=None,
+        incoming_topic,
```

Comment:
why was all the other options deleted?

Reply:
they come in kwargs, so it makes more sense to retrieve them from kwargs.get

Reply:
They are default `KafkaConsumer` arguments, they are already "included" as `kwargs`. The only `kwarg`-able argument that is mandatory for us is the `incoming_topic`, so I removed the default value and kept it.

If we want to use those arguments, we just need to put them in the YAML configuration file and they will be passed to the `KafkaConsumer`